app/controllers/providers/identify_types_of_outgoings_controller.rb

```diff
@@ -1,6 +1,42 @@
 module Providers
   class IdentifyTypesOfOutgoingsController < ProviderBaseController
-    def show
+    before_action :authorize_legal_aid_application
+
+    def show; end
+
+    def update
+      return continue_or_draft if none_selected? || transactions_added
```

Comment:
If the user selects "save as draft" without selecting anything, should they see an error? I assumed not. This version of this line behaves that way:
```ruby
return continue_or_draft if draft_selected? || none_selected? || transactions_added
```

Reply:
Yes, I agree with you. I'll add that.